server/sandbox_remove.go

```diff
@@ -18,7 +18,9 @@ func (s *Server) RemovePodSandbox(ctx context.Context, req *pb.RemovePodSandboxR
 	logrus.Debugf("RemovePodSandboxRequest %+v", req)
 	sb, err := s.getPodSandboxFromRequest(req)
 	if err != nil {
-		return nil, err
```

Comment:
This seems wrong as `getPodSandboxFromRequest` can return a legit error when the `req` doesn't contain a sandbox id at all. We should differentiate between "no sandbox id in req" (where we fail) and "can't find sandbox because it has been already removed" (where we pass).